### ISSUE TITLE
Is it possible to have multiple datasources in the HTML Value Card Widget ?

### ISSUE THREAD

**Issue by lucasKapf** [thingsboard/thingsboard#5662]
Hello,

I would like to know if it is possible to add multiple datasources in the HTML value card widget : 
I want to display the client and its information, and display the number of assets that this client contains on the same widget.
Is it possible ?

Best regards,

Lucas

**Comment by lucasKapf** [comment]
Hi @dhaval2017 thanks for your quick answer.  I'm gonna do this

**Comment by lucasKapf** [comment]
So I am trying to create a widget similar to the HTML Value Card Widget but with multiple datasources.
For the moment I started with the original HTML Value Card Widget Code :  
I just changed the maxDataSources value to 2 and the singleEntity to false inside the self.typeParameters function.
But when I use my custom widget, only the first datasource is taken into account. 
I think that I have to modify/add more code inside my custom widget but I don't know where to start.
